circle.yml

```diff
@@ -1,48 +0,0 @@
-experimental:
-  notify:
-    branches:
-      only:
-        - master
-        - /^__release-.*/
-
-machine:
-  xcode:
-    version: 7.3
-
-dependencies:
-  cache_directories:
-    - ".eslintcache"
-    - "node_modules"
-    - "~/.electron"
-    - "~/Library/Caches/Yarn/v1"
-
-  pre:
-    - brew update
-    # uninstall Node 6 which is shipped by default on Circle
-    - brew uninstall node
-    # install latest Node LTS (8.11.* as of writing)
-    - brew install node@8 --force
-    # everything is terrible, but we must march on
-    - brew link --overwrite node@8 --force
-    # workaround for yarn installation issue
-    - curl -o- -L https://yarnpkg.com/install.sh | TERM=xterm bash -s -- --version 1.5.1
-
-  override:
-    - yarn install --force
-
-compile:
-  override:
-    - yarn lint
-    - yarn validate-changelog
-    - yarn build:prod
-
-test:
-  override:
-    - yarn test:setup
-    - yarn test
-
-deployment:
-  production:
-    branch: /^__release-.*/
-    commands:
-      - yarn run publish
```

Comment:
This step here hasn't been ported over to the new config format - any context on why that's the case?

Reply:
The context is that I didn't drink enough coffee and have missed this step 😅